src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+    final int TWELVE = 12;
+    final int MINUS_ONE = -1;
+    final int ELEVEN = 11;
```

Comment:
This constant is related to the capacity of numbers. Changing only one leads to an error (inconsistency).